index.html

```diff
@@ -4,50 +4,158 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <title>Nguyễn Công Việt</title>
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
   <link rel="stylesheet" href="/assets/style.css">
   <style>
     html body {
       margin: 0;
     }
 
-    .navbar ul{
-      margin: 15px;
-      display: flex;
-      float: left;
+    .navbar .menu{
```

Comment:
Chú ý thêm dấu cách để trông code đẹp gọn.
```suggestion
    .navbar .menu {
```